zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageConfiguration.java

```diff
@@ -137,6 +144,21 @@ SessionProtocol esSessionProtocol(ZipkinElasticsearchStorageProperties es) {
     };
   }
 
+  @Bean @Qualifier(QUALIFIER) @Conditional(BasicAuthRequired.class)
+  BasicCredentials dynamicElasticsearchBasicCredentials(ZipkinElasticsearchStorageProperties es) {
```

Comment:
nit: i would call this method just `basicCredentials` since at this point they are not dynamic per se, they become dynamic when you hook them up to the `DynamicCredentialsFileLoader`

Reply:
make sense. have updated the method name